pom.xml

```diff
@@ -95,7 +95,7 @@
     <url>https://github.com/sinch/sinch-java-sms</url>
     <connection>scm:git:git://github.com/sinch/sinch-java-sms.git</connection>
     <developerConnection>scm:git:git@github.com:sinch/sinch-java-sms.git</developerConnection>
-    <tag>HEAD</tag>
+    <tag>v2.2.0</tag>
```

Comment:
Is this correct? I would have guessed that this should be `v2.2.1`

Reply:
I suppose it should be HEAD